lib/content-services/social/like.component.scss

```diff
@@ -40,3 +40,11 @@
     }
 
 }
+
+[dir='rtl'] .adf-like-container {
+    display: flex;
```

Comment:
Why does this container gets to be flex only when direction is rtl? Also, there are a lot of micro styling for each item with floats and paddings and colours that can be reused

I think this file can be boiled down to something like 
```scss
@mixin selected {
  color: #2196f3; // we should use css variable or apply mat-theme
}

@mixin unselected {
  color: #808080; // we should use css variable or apply mat-theme
}

.adf-like-container {
    overflow: hidden;
    width: 100%;
    display: flex;
    align-items: center;

    .item {
        padding: 0 2px;
    }

    .item--cursor {
        cursor: pointer;
    }

    .item--selected {
         @include selected;
         &:hover {
              @include unselected;
         }
    }

    .item--unselected {
        @include unselected;
        &:hover {
           @include selected;
        }
    }
}
```

and the template should be something like 

```html

<div id="adf-like-container" class="adf-like-container">
    <div class="item item--cursor">
        <span id="adf-like-{{nodeId}}" [ngClass]="{'item--selected': isLike ,'item--unselected': !isLike  }"
              (click)="likeClick()">
            <mat-icon>thumb_up</mat-icon>
        </span>
    </div>
    <div id="adf-like-counter" class="item">{{likesCounter}}</div>
    <div class="item" *ngIf="likesCounter<=1">Like</div>
    <div class="item" *ngIf="likesCounter>1">Likes</div>
</div>
```

`item` is just for example, you should use some BEM here and you wont need to target rtl or ltr